app/controllers/api/LICENSE

```diff
@@ -0,0 +1,19 @@
+Copyright (c) 2012 Brian Ploetz (bploetz@gmail.com)
```

Comment:
Ah cheez. See, this is why I got angry with copying sources in...

Unfortunately, we cannot simply add LICENSE files to those folders since we also have done a lot of work and our stuff is AGPL. So now, we have to go through all files [nov/openid_connect_sample](https://github.com/nov/openid_connect_sample), check if we are still using that exact file and add a license header (inside a comment block) to all copied files.

Sorry for being picky here, but we should better not make licensing mistakes...